sportsref/nfl/teams.py

```diff
@@ -240,6 +240,49 @@ def team_stats(self, year):
         return df.ix[df.player_id == 'Team Stats'].iloc[0]
 
     @sportsref.decorators.memoize
+    def injuries(self, year):
+        """ This gets the injuries from pro-football-reference for each team
+
+        :param year: year to query
+        :return: the pfrid and injury status per week for the year
+        """
+        # set link and table_name and then get the pyquery table
+        link = sportsref.nfl.BASE_URL + \
+               '/teams/{}/{}_injuries.htm'.format(self.teamID, str(year))
+        doc = pq(sportsref.utils.get_html(link))
```

Comment:
You'd get those as `self.get_year_doc('2012_injuries')` and `self.get_year_doc('2012-snap-counts')`. See the way they're used in the roster function. It just wraps those first two lines in a function, but those functions become tedious when they all do the same thing.